src/components/legacy/index.js

```diff
@@ -8,4 +8,5 @@ exports.makeRenderable = exports.renderable = require("../../runtime/renderable"
 
 // server only
 exports.writeInitWidgetsCode = modernMarko.writeInitComponentsCode;
-exports.getRenderedWidgets = modernMarko.getRenderedComponents;
+exports.getRenderedWidgets = exports.getRenderedWidgetIds =
```

Comment:
Add deprecation here.

Reply:
Since this file is what you get from `require('marko-widgets')` I don't think we need to explicitly deprecate any specific apis since the whole package should be considered deprecated.